src/components/Form/UploadDocumentForm.jsx

```diff
@@ -56,20 +57,13 @@ const UploadDocumentForm = () => {
   const handleDocUpload = async (e) => {
     e.preventDefault();
 
-    if (!docType) {
```

Comment:
Other recent changes (not in this PR) block form submit if this  field isn't filled out, so we can't hit this code branch. Removing.